openc3-cosmos-script-runner-api/scripts/run_suite_analysis.py

```diff
@@ -45,8 +45,6 @@ def load_utility(procedure_name):
     return False
 
 
-require_utility = load_utility
```

Comment:
This is breaking.  There is no harm in supporting both require_utility and load_utility.